site/docs/pages/token/introduction.mdx

```diff
@@ -11,4 +14,23 @@ Coming soon!
 
 ## Components
 
+The available components are:
+
+- [`<TokenChip />`](/token/token-chip): Small button component for a given token.
+- [`<TokenRow />`](/token/token-row): Row component for a given token.
+- [`<TokenSearch />`](/token/token-search): Search component to search by name, symbol and address for a given list of tokens.
+
```

Comment:
Btw you want to show how is built as well.

Similar to https://onchainkit.xyz/identity/introduction

Reply:
Added code snippet. It's a tad bit long.

Reply:
it's ok we can think how to clean up that later.